### PR TITLE
Added tests for pull #943

### PR DESCRIPTION
**Pull request by cossio**
https://github.com/JuliaOpt/JuMP.jl/pull/943
https://github.com/JuliaOpt/JuMP.jl/issues/940

### CODE REVIEW

File: test/operator.jl [hunk 1]
```diff
@@ -495,6 +495,17 @@ const sub2 = JuMP.repl[:sub2]
         anys[1] = 10
         anys[2] = 20 + x
         @test dot(floats, anys) == 10 + 40 + 2x
+            
+        # https://github.com/JuliaOpt/JuMP.jl/pull/943
+        pull943 = Model()
+        @variable(pull943, x[1 : 10^6]);
+        setvalue.(x, rand(n))
```
Comment (joehuchette) [inline test/operator.jl]: What is ``n``?

Comment (cossio) [inline test/operator.jl]: Sorry, that's 10^6. Fixed it.

File: test/operator.jl [hunk 2]
```diff
@@ -495,6 +495,17 @@ const sub2 = JuMP.repl[:sub2]
         anys[1] = 10
         anys[2] = 20 + x
         @test dot(floats, anys) == 10 + 40 + 2x
+            
+        # https://github.com/JuliaOpt/JuMP.jl/pull/943
+        pull943 = Model()
+        @variable(pull943, x[1 : 10^6]);
+        setvalue.(x, rand(10^6))
+        setvalue.(y, 1 : 10^6)
```
Comment (joehuchette) [inline test/operator.jl]: What is ``y``? This part is failing Travis.

Comment (cossio) [inline test/operator.jl]: Sorry again. I copy and pasted from a version that had other var names.
Fixed it.

**Comment by cossio** [comment]
@joehuchette  Do you think having random data in the test is a good idea? It's been worrying me. I could replace that with some fixed data.

**Comment by mlubin** [comment]
> Do you think having random data in the test is a good idea?

No, it's not a good idea.

**Comment by mlubin** [comment]
There's no need to test with ``10^6`` variables, three would be just fine.

**Comment by joehuchette** [comment]
That wouldn't trigger the warning in the case of a regression, though.

**Comment by mlubin** [comment]
Ah, right

**Comment by cossio** [comment]
@mlubin Using fixed data now.

**Comment by joehuchette** [comment]
Thanks, @cossio.

In the future, it would be preferable to attach the tests to the PR which lead to them. Learning to use git is a good investment of time :)

**Pull request merged**